api.d.ts

```diff
@@ -14,9 +14,7 @@ export interface OutgoingPullRequest extends PullRequest {
   reviewRequests: string[];
 }
 
-export interface DashResponse {
-  outgoingPrs: OutgoingPullRequest[];
-}
+export interface DashResponse { outgoingPrs: OutgoingPullRequest[]; }
```

Comment:
Is this the auto formatter? Looks odd

Reply:
Yep, it's the auto formatter. It wasn't configured to format this file before this PR.